api/v1alpha1/apidefinition_types.go

```diff
@@ -627,8 +627,8 @@ type APIDefinitionSpec struct {
 	//DisableRateLimit       bool                `json:"disable_rate_limit"`
 	//DisableQuota           bool                `json:"disable_quota"`
 
-	CustomMiddleware MiddlewareSection `json:"custom_middleware,omitempty"`
-	//CustomMiddlewareBundle string              `json:"custom_middleware_bundle"`
+	CustomMiddleware       MiddlewareSection `json:"custom_middleware,omitempty"`
+	CustomMiddlewareBundle string            `json:"custom_middleware_bundle"`
```

Comment:
Would be good to have some documentation for these fields - as we are here. What does `CustomMiddleware` and `CustomMiddlewareBundle` do? How do I use it? is there an example? 

Also, why not `omitempty`? ``json:"custom_middleware_bundle,omitempty"``